core/templates/dev/head/components/exploration_embed_button_modal.html

```diff
@@ -9,17 +9,8 @@
     </p>
 
     <accordion close-others="true">
-      <accordion-group heading="If your website allows JavaScript" is-open="false">
-        <div class="oppia-embed-modal-code" ng-click="selectText($event)">
-&lt;oppia oppia-id="<[explorationId]>" src="<[serverName]>"&gt;&lt;/oppia&gt;
-&lt;script src="https://cdn.jsdelivr.net/oppia/0.0.1/oppia-player.min.js"&gt;&lt;/script&gt;
-        </div>
-      </accordion-group>
-      <accordion-group heading="If your website does not allow JavaScript" is-open="false" style="margin-top: 3px;">
-        <div class="oppia-embed-modal-code" ng-click="selectText($event)">&lt;iframe src="<[serverName]>/explore/<[explorationId]>?iframed=true" width="700" height="1000"&gt;</div>
-        <p>
-          <em>Note: explorations embedded using iframes will not have automatic scrolling and expanding on your webpage.</em>
-        </p>
+      <accordion-group heading="If your website allows Iframe" is-open="false" style="margin-top: 3px;">
```

Comment:
As mentioned previously, let's not change this file at all yet, until the new UI is finished and we have established a procedure for migrating existing users from the old UI to the new UI. Could you please revert all the changes to this file?